wiki/Guides/Livestreaming_osu!/en.md

```diff
@@ -0,0 +1,65 @@
+# Livestreaming osu!
+
+This guide will show you how to livestream osu! on Twitch using [OBS Studio](https://obsproject.com/ "Open Broadcaster Software"). Doing so is relatively simple, although it might require a reliable computer able to both run and livestream osu!.
+
+## Twitch
+
+If you have not already, sign up for a [Twitch account](https://www.twitch.tv/signup "Twitch").
+
+Now you will need to setup your account for livestreaming osu!. From the main page, click on your avatar in the top right and select `Creator Dashboard` from the list.
+
+![](img/dashboard.png "Entering the Creator Dashboard")
+
+Creator Dashboard is the place where you can configure your stream information, chat moderation, profile info, and more. For now, we will focus on stream information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
+
+A dialog will open for you to enter your stream's information. You can fill the fields however you want; however, the title should comprehend what the stream will be about and also be inviting for people viewing the list of available streams. The category must be set to the game you are playing at the moment, which would be `osu!`.
+
+![](img/stream-info.png "Editing your stream's information")
+
+---
```

Comment:
somewhere around here you should mention that the category is mirrored on the website as well and is updated once in a while (think once in 1 or 5 minutes): https://osu.ppy.sh/community/livestreams